app/views/home/index.html.erb

```diff
@@ -9,24 +9,24 @@
   <%= render 'counter', locals: { number: @num_devices, text: 'servers entered' } %>
 
   <div class="cl-card flex-column-centred">
```

Comment:
If you could that that'd be great, thanks.

I suppose that group names can TECHNICALLY have infinite length, but it'll take an exponentially increasing number of groups for each new character. I feel like it's out of scope to assume that we'll ever have a `Metal-Group16578915647589164716598714657812-GBR` but maybe 4 or 5 characters would be sufficient? i.e. supporting `Metal-Group12345-GBR` since adding thousands of nodes with slightly different specs certainly seems possible.

Reply:
I realised that instance types might also be long. So I've increased the width of the home leaderboard but also made sure that if the name is too long, the end will be cut off with ellipses. I've also done this for the full leaderboard column. The first thing to be cut off will be location, which is also shown elsewhere so hopefully not too much info will be lost.

Reply:
Something Stu noticed last week was that given the new group names, the group leaderboard columns might need to be reworked - the current group naming scheme is PLATFORM-(INSTANCE TYPE/GROUP NO.)-LOCATION, which makes the following `Platform` and `Location` fields somewhat irrelevant:
![image](https://github.com/openflighthpc/carbon-leaderboard/assets/57297493/c866a5a7-e5f9-4c6b-be01-c8d32d3ce113)

Especially since the alces nodes are displayed as OpenStack

Reply:
So potentially we could assign a lot more space to the group names without it being any issue, or we could add some more columns to the table (e.g. a flag column because it looks nice)

Reply:
Alternatively, we could not show the group name column, and just show the fields in separate columns. That might be more easily readable and more attractive than just showing all the data in one string

Flag column sounds good either way :+1:

Reply:
As we're now planning to icon-ify the platform and location columns, shall we leave the columns as they are for now and make the changes in a different PR? Then we can use the icons added in the server page PR

Reply:
I think that's a good idea - I'm going to make a data migration for this branch to convert between g and kg for carbon values, aside from that I'm keen to just get it merged since it should end up being the last significant set of changes to the backend (hopefully)